src/lib/canvas/canvas.js

```diff
@@ -1,70 +1,63 @@
+import 'typeface-montserrat';
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import Grid from './grid';
 import { defaultFontFamily } from '../constants';
-import { guid } from '../utils';
+import { guid, isAnyNodeWithTabs, isAllNodesWithTabs } from './utils';
 
 const Container = styled.div`
   font-family: ${defaultFontFamily};
+  width: ${props => props.width}px;
+  height: ${props => props.height}px;
   font-style: regular;
   font-size: 13px;
   background: #515151;
   color: #A9A9A9;
-  width: 100%;
-  height: 100%;
   min-width: 500px;
   min-height: 100px;
   padding 8px;
 `;
 
 class Canvas extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      layout: {
-        id: guid(),
-        type: 'element',
-        tabs: []
-      }
-    };
-
-    this.handleUpdate = this.handleUpdate.bind(this);
-  }
-
-  handleUpdate(layout) {
-    this.setState({ layout });
-  }
-  
   handleDragStart(e) {
     e.dataTransfer.setData('creatorId', e.target.id);
   }
 
   componentDidMount() {
-    for (const el of document.getElementById(this.props.creatorListId).children) {
-      el.addEventListener('dragstart', this.handleDragStart);
+    const list = document.getElementById(this.props.buildersListId)
+    if (list) {
+      for (const el of list.children) {
+        el.addEventListener('dragstart', this.handleDragStart);
+      }
     }
   }
-  
+
   componentWillUnmount() {
-    for (const el of document.getElementById(this.props.creatorListId).children) {
-      el.removeEventListener('dragstart', this.handleDragStart);
+    const list = document.getElementById(this.props.buildersListId);
```

Comment:
much common with the logic in componentDidMount - suggest to create a function and call it in both places